music/src/core/player.ts

```diff
@@ -400,7 +417,10 @@ export class Player extends BasePlayer {
       bass.volume.value = 5;
       this.synths.set(instrument, bass);
     } else {
-      this.synths.set(instrument, new Tone.PolySynth(10).toMaster());
+      // If no synth is present for the proviced voice, set the voice to the
+      // default synth instrument.
+      const defaultSynth = this.synths.get(this.DEFAULT_SYNTH_INSTRUMENT);
```

Comment:
wouldn't we have an issue here since we could only ever have 2 instruments: bass or the default synth.

let's say I call `getSynth(1)` and then `getSynth(2)` I would get the same default synth in both cases which I don't think is the expected behavior.

Reply:
Yeah, as I mentioned in the description this is where things get a bit wonky. The previous behavior was to instantiate a new synth on-demand during playback, which both causes glitches in tone 0.13 and is a bit weird, IMO. The correct way to do this would be to set voices explicitly. I'm happy to move towards doing this, but it's a bigger refactor - figured I'd start with the smallest footprint.

Reply:
Also - wow, not sure how I didn't realize there's a whole other massive PR with a lot more context for this same issue. Closing in favor of continuing the conversation there.